wolfssl/wolfcrypt/port/Renesas/renesas-sce-crypt.h

```diff
@@ -34,10 +34,30 @@ extern "C" {
 
 #define MAX_SCE_CBINDEX 5
 
+typedef void* renesas_sce_wrappedkey;
+
+/* related to TLS */
+struct sce_flags1 {
+    uint8_t pk_key_set:1;
+    uint8_t session_key_set:1;
+};
+
+/* Crypt Only */
+struct sce_flags2 {
```

Comment:
Should this struct name be something like `sce_flags_crypto` instead, to make it more clear what it is.  I could see `flags2` being unclear for another developer.

Reply:
Agree to you. How about `sce_keyflgs_cryt`?